### PR TITLE
Added more UI display for Player Controller

### PR DESCRIPTION
**Pull request by uchennaponuigbo**
Added current exp, exp for next level, and pebble count. And I found and fixed a display issue where Kento's health would display negative numbers if the damage taken is greater than his remaining HP.

### CODE REVIEW

File: the-lost-penguin/Assets/Scripts/PlayerStats.cs [hunk 1]
```diff
@@ -71,32 +79,37 @@ public void AddExp(int exp)
...
         //update text on screen
         kentoLevel++;
-        expForNextLevel = expToNextLevelList[kentoLevel];
+        expForNextLevel = expToNextLevelList[kentoLevel - 1];
+        UIController.instance.UpdateExpLevel();
     }
 
     private void IncreaseStats()
     {
         bellySlide++; //slide distance or damage?
```
Comment (Ghop11) [inline the-lost-penguin/Assets/Scripts/PlayerStats.cs]: I do not think we are doing belly slides anymore. We need to discuss this.

File: the-lost-penguin/Assets/Scripts/PlayerHealth.cs [hunk 2]
```diff
@@ -49,11 +48,17 @@ public void DamagePlayer(int value = 10)
...
         
             if (currentHealth - value <= 0)
-            {
+            {   
                 // Kento is knocked out
                 KnockedOut();
             }
+            
             currentHealth -= value;
+            //found a display issue where Kento's Health goes into the negatives
```
Comment (Ghop11) [inline the-lost-penguin/Assets/Scripts/PlayerHealth.cs:57]: Good addition

**Review by Ghop11 — APPROVED** [review]
Looks good to me. I left some comments you can take a look at as well.

File: the-lost-penguin/Assets/Scripts/PlayerStats.cs (hunk 1 above)
Comment (uchennaponuigbo) [reply the-lost-penguin/Assets/Scripts/PlayerStats.cs]: alright

**Review by das-mittel — APPROVED** [review]
Reviewed and approved!

**Review by uchennaponuigbo — COMMENTED** [review]
done

**Pull request merged**